### ISSUE TITLE
Ligatures for #( and #{ are very difficult to read

### ISSUE THREAD

**Issue by gilbertohasnofb** [tonsky/FiraCode#1184]
I am working on a Python project whose documentation often include LilyPond code, which contains Scheme-like syntax. The ligatures for `#(` and `#{` are very difficult to read. See red arrows below (green arrow shows that `#}` behaves fine):

![image](https://user-images.githubusercontent.com/5700795/104822832-5fe49f80-583d-11eb-80cf-e3925ae40d2a.png)

Is this the intended output?

**Comment by RubenVerg** [comment]
It is, and as a frequent LilyPond author I can read those just fine. that's my opinion

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by gilbertohasnofb** [comment]
If that is by design, then what is the purpose of it? Most ligatures in FiraCode seem to be implemented either  forimproved readability (e.g. `++`, `..`, `:=`, `||`, etc.) or for displaying complex combinations of characters (e.g. `->`, `!=`, `<=`, `>=`, `/=`, etc.), making the meaning of these combinations clearer. I can of course _read_ FiraCode's `#{` and `#(`, but the readability is, for me at least, substantially poorer than with most other typefaces. At a small font size (such as in my screenshot above), the readability is quite poor. The combination `#{` is particularly problematic, since the part that sticks out at the middle of the curly bracket is just 2 pixels long and is surrounded by the two extended horizontal lines from the hash sign. When I look at the image below, I need to squint my eyes and it takes me a second to figure out which is which.

![image](https://user-images.githubusercontent.com/5700795/104847047-bdc4c600-58d5-11eb-8ae5-23c11de0c3df.png)

**Comment by tonsky** [comment]
I agree that this might be hard to read. I am not very proud of these particular ligatures. I am not opposed to removing `#{` at least.

They were initially added for Clojure, with the idea to show that these two symbols produce a single logical unit, thus it’s drawn as a single symbol

**Comment by RubenVerg** [comment]
#( has rounded borders, #{ looks more like a rounded square. They're clear as-is. Then, I'm not opposing against a stylistic set to disable them, but I feel like they're easily discernable

________________________________

**Comment by gilbertohasnofb** [comment]
@tonsky Thanks for the reply and for the awesome project. I completely understand the reasoning behind these, it's just unfortunate they are so difficult to read at a typical font size. 

> I am not opposed to removing #{ at least.

Personally, I would remove it from both for consistency. My 2 cents.

**Comment by RubenVerg** [comment]
I'm kinda on the opposite route, add support for `#}` as well
